ts-sdk/whirlpool/tests/utils/tokenExtensions.ts

```diff
@@ -30,7 +31,7 @@ export async function setupAtaTE(
   instructions.push(
     getCreateAssociatedTokenIdempotentInstruction({
       mint,
-      owner: signer.address,
+      owner: owner.address,
       ata: ata[0],
       payer: signer,
```

Comment:
If I set payer to owner, error is occurred because owner doesn't have enough balance.